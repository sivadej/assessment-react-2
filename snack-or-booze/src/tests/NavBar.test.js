import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';
import NavBar from '../NavBar';

it('renders without breaking', ()=> {
  render(<BrowserRouter><NavBar/></BrowserRouter>);
});

it('matches snapshot', ()=> {
  const { asFragment } = render(<BrowserRouter><NavBar/></BrowserRouter>);
  expect(asFragment()).toMatchSnapshot();
});