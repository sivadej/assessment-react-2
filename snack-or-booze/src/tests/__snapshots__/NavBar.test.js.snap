// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <nav
      class="navbar navbar-expand-md"
    >
      <a
        aria-current="page"
        class="navbar-brand active"
        href="/"
      >
        Snack or Booze
      </a>
      <ul
        class="ml-auto navbar-nav"
      >
        <li
          class="nav-item"
        >
          <a
            href="/snacks"
          >
            Snacks
          </a>
        </li>
        <li
          class="nav-item"
        >
          <a
            href="/drinks"
          >
            Drinks
          </a>
        </li>
      </ul>
    </nav>
  </div>
</DocumentFragment>
`;
